src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -537,6 +544,24 @@ public GetYourTranscriptsResponse getYourTranscripts(Integer userId) {
     }
 
     @Override
+    public TranscriptionTranscriberCountsResponse getTranscriptionTranscriberCounts(Integer userId) {
+
+        UserAccountEntity user = userAccountRepository.findByRoleAndUserId(TRANSCRIBER.getId(), userId);
+        if (user == null) {
+            throw new DartsApiException(USER_NOT_TRANSCRIBER);
+        }
+
+        List<Integer> courthouseIds = transcriberTranscriptsQuery.getAuthorisedCourthouses(userId, 4);
```

Comment:
There is an enum we can use here to try and make it more readable for our standing data: `SecurityRoleEnum#TRANSCRIBER`

Reply:
used it